whois-nrtm/src/main/java/net/ripe/db/whois/nrtm/NrtmServerChannelInitializer.java

```diff
@@ -34,25 +41,36 @@
     private final NrtmQueryHandlerFactory nrtmQueryHandlerFactory;
     private final NrtmConnectionPerIpLimitHandler nrtmConnectionPerIpLimitHandler;
     private final NrtmAclLimitHandler nrtmAclLimitHandler;
+    private final Optional<ProxyProtocolChannelHandler> proxyProtocolChannelHandler;
 
     protected NrtmServerChannelInitializer(final NrtmChannelsRegistry nrtmChannelsRegistry,
                                                final NrtmExceptionHandler exceptionHandler,
                                                final MaintenanceHandler maintenanceHandler,
                                                final NrtmQueryHandlerFactory nrtmQueryHandlerFactory,
                                                final NrtmAclLimitHandler nrtmAclLimitHandler,
-                                               final NrtmConnectionPerIpLimitHandler nrtmConnectionPerIpLimitHandler) {
+                                               final NrtmConnectionPerIpLimitHandler nrtmConnectionPerIpLimitHandler,
+                                               final @Value("${proxy.protocol.enabled:false}") boolean proxyProtocolEnabled) {
         this.nrtmChannelsRegistry = nrtmChannelsRegistry;
         this.exceptionHandler = exceptionHandler;
         this.maintenanceHandler = maintenanceHandler;
         this.nrtmQueryHandlerFactory = nrtmQueryHandlerFactory;
         this.nrtmConnectionPerIpLimitHandler = nrtmConnectionPerIpLimitHandler;
         this.nrtmAclLimitHandler = nrtmAclLimitHandler;
+
+        if (proxyProtocolEnabled) {
```

Comment:
This [comment](https://github.com/RIPE-NCC/whois/pull/780/files#r590592854) also applies here

Reply:
Done.